pandas/core/generic.py

```diff
@@ -11805,6 +11805,7 @@ def _logical_func(
 
     def any(
         self,
+        *,
```

Comment:
Should these be in your follow up PR? Does it change the current behavior of e.g. `DataFrame/Series.any()`?

Reply:
I would be fine having those changes: NDFrame is private and all those methods are overwritten by `DataFrame` and `Series`, so these changes are not user-facing. Let me revert them to keep it less controversial.

Reply:
If you don't mind, I will revert this change back - see failing mypy&stubtest